requirements.md

```diff
@@ -24,36 +24,29 @@
 
 ## Use Cases
 
-1. Use Case Name (Should match functional requirement name)
-- **Pre-condition:** <can be a list or short description> Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua.
+1. Open user home page (user can see messages of users they follow)
+- **Pre-condition:** 
+  -user is logged in
```

Comment:
Can you move this to "Trigger" instead as "User logins in with correct credentials", and maybe add "User is on the login page" as "Pre-condition"? This will make the use case look simpler.